packages/avatar/contracts/nft-collection/NFTCollection.sol

```diff
@@ -737,6 +764,7 @@ INFTCollection
 
     /**
      * @notice the total amount of waves configured till now
+     * @return the wave count
```

Comment:
I feel like the notice text should also be in the @return text, maybe the notice should be like "Returns ..":